app/views/pages/netlify_config.yaml.erb

```diff
@@ -132,6 +135,37 @@ collections:
               Order in which this item should be sorted in navigation menus.
               Higher numbers sort after lower numbers.
               If two pages have the same Order, they're sorted alphabetically by title.
+      - label: "Lifecycle"
+        name: "lifecycle"
+        widget: object
+        collapsed: true
+        summary: "Expires: {{fields.expires_at | default('never')}}"
```

Comment:
[question] should the summary also display info about whether "replaced by page" is set?

Reply:
Yes, let me see how that ends up looking (there aren't a ton of formatting controls for the `summary` field)